tests/integrations/openai_tracker/test_sync.py

```diff
@@ -0,0 +1,177 @@
+import os
+import time
+import uuid
+
+import pymysql  # type: ignore
+from openai import OpenAI
+
+from greptimeai import openai_patcher  # type: ignore
+
+
+class LlmTrace(object):
+    table_name = "llm_traces_preview_v01"
+
+    trace_id: str
+    span_id: str
+    parent_span_id: str
+    resource_attributes: str
+    scope_name: str
+    scope_version: str
+    scope_attributes: str
+    trace_state: str
+    span_name: str
+    span_kind: str
+    span_status_code: str
+    span_status_message: str
+    span_attributes: str
+    span_events: str
+    span_links: str
+    start: float
+    end: float
+    user_id: str
+    model: str
+    prompt_tokens: int
+    prompt_cost: float
+    completion_tokens: int
+    completion_cost: float
+    greptime_value: str
+    greptime_timestamp: float
+
+
+class LlmPromptToken(object):
+    table_name = "llm_prompt_tokens"
+
+    telemetry_sdk_language: str
+    telemetry_sdk_name: str
+    telemetry_sdk_version: str
+    service_name: str
+    span_name: str
+    model: str
+    greptime_value: str
+    greptime_timestamp: float
+
+
+class LlmPromptTokenCost(object):
+    table_name = "llm_prompt_tokens_cost"
+
+    telemetry_sdk_language: str
+    telemetry_sdk_name: str
+    telemetry_sdk_version: str
+    service_name: str
+    span_name: str
+    model: str
+    greptime_value: str
+    greptime_timestamp: float
+
+
+class LlmCompletionToken(object):
+    table_name = "llm_completion_tokens"
+
+    telemetry_sdk_language: str
+    telemetry_sdk_name: str
+    telemetry_sdk_version: str
+    service_name: str
+    span_name: str
+    model: str
+    greptime_value: str
+    greptime_timestamp: float
+
+
+class LlmCompletionTokenCost(object):
+    table_name = "llm_completion_tokens_cost"
+
+    telemetry_sdk_language: str
+    telemetry_sdk_name: str
+    telemetry_sdk_version: str
+    service_name: str
+    span_name: str
+    model: str
+    greptime_value: str
+    greptime_timestamp: float
+
+
+class Number(object):
+    table_name = "number"
+
+    number: int
+
+
+db = pymysql.connect(
+    host=os.getenv("GREPTIMEAI_HOST"),
+    user=os.getenv("GREPTIMEAI_USERNAME"),
+    passwd=os.getenv("GREPTIMEAI_PASSWORD"),
+    port=4002,
+    db=os.getenv("GREPTIMEAI_DATABASE"),
+)
```

Comment:
Is it possible to make the mysql logic into other module? because langchain will use it too.

Reply:
I was thinking the same thing, but I had problems importing packages in the tests directory. Packages that import greptimeai can only import packages under src.

Reply:
done